WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -1251,8 +1251,14 @@ private extension AztecPostViewController {
             }
         }
 
-        alert.addDefaultActionWithTitle(MoreSheetAlert.postSettingsTitle) { [unowned self] _ in
-            self.displayPostSettings()
+        if let _ = post as? Post {
```

Comment:
We could remove a little duplication here by initializing a string based on the post type

```
let settingsTitle = (post is Post) ? MoreSheetAlert.postSettingsTitle : MoreSheetAlert.pageSettingsTitle
alert.addDefaultActionWithTitle(settingsTitle) { [weak self] _ in
    self?.displayPostSettings()
}
```